package.json

```diff
@@ -33,6 +33,7 @@
   "devDependencies": {
     "@types/chai": "^4.1.0",
     "@types/mocha": "^5.2.0",
+    "@webcomponents/shadycss": "^1.5.2",
```

Comment:
why do we need this?

Reply:
Okay it turns out we didn't need this.  My recollection was I added this because I saw a script reference to it, but I must have been mistaken because I don't see that direct reference anywhere.  Removed.

Reply:
Oh wait.  Now I discover why I added this.  Here's the network panel when I'm running tests with WCT:

![screen shot 2018-10-22 at 1 17 39 pm](https://user-images.githubusercontent.com/578/47316891-1032b180-d5fd-11e8-9225-e4a6748804ff.png)

Reply:
With shadycss added as dependency, then it's all 200s:

![screen shot 2018-10-22 at 1 43 19 pm](https://user-images.githubusercontent.com/578/47318275-8258c580-d600-11e8-9658-9714b87584c8.png)

Thoughts? @justinfagnani

Reply:
Oh derp so yeah totally this line loads it https://github.com/Polymer/lit-html/pull/577/files#diff-ba8440caa7003220705be8efef87282bR16

`<script src="../node_modules/@webcomponents/shadycss/apply-shim.min.js"></script>`